Makefile

```diff
@@ -19,10 +19,10 @@ ifndef $(GOPATH)
 endif
 
 check: diffproviders.txt
-	CGO_ENABLED=0 go test -v $(PKG)/cmd/... $(PKG)/pkg/...
-	
 	curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(shell go env GOPATH)/bin v1.23.8
```

Comment:
This is functionally a minor change but it allows syntax errors to be caught and surfaced before the go tests. 

It's helpful when debugging. We shouldn't run `go test` if there is a lint failure.